behat/symfony2-extension/2.1/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "copy-from-recipe": {
+        "bin/": "%BIN_DIR%",
+        "behat.yaml.dist": "behat.yaml.dist"
```

Comment:
Can this configuration be based on `.yml` files not `.yaml` (it does not pass validation)?  For Behat [`v3.3.0`](https://github.com/Behat/Behat/blob/v3.3.0/src/Behat/Behat/ApplicationFactory.php#L116) it checks only `.yml` files. As I can see it's improved on [`maser`](https://github.com/Behat/Behat/blob/master/src/Behat/Behat/ApplicationFactory.php#L116)